modules/motus/downloaddb/main.nf

```diff
@@ -0,0 +1,39 @@
+process MOTUS_DOWNLOADDB {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::motus=3.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/motus:3.0.1--pyhdfd78af_0':
+        'quay.io/biocontainers/motus:3.0.1--pyhdfd78af_0' }"
+
+    input:
+    path motus_downloaddb
+
+    output:
+    path "db_mOTU"                , emit: db
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    ## must copy file to working directory,
+    ## otherwise the reference_db will be download to bin folder
+    ## other than current directory
+    cp $motus_downloaddb ${motus_downloaddb.simpleName}_copy.py
+    python ${motus_downloaddb.simpleName}_copy.py \\
+        -t $task.cpus
+    ## clean up
+    rm ${motus_downloaddb.simpleName}_copy.py
```

Comment:
You don't need to delete these yourselve, I would leave it up to nextflow to deal with this